src/Controller/Api/FileManager.php

```diff
@@ -155,21 +161,19 @@ private function deleteFile(File $file, Fileset $fileset, Datastore $datastore)
             }
         } elseif ($file->getStatus() === File::FILE_DONE) {
             $file->setStatus(File::FILE_DELETED);
-            $filePath = $file->getPhysicalFilePath();
-            $newFilePath = $filePath . Datastore::MARK_FILE_AS_DELETED;
-            $newFilePath = $datastore->renameFile($filePath, $newFilePath, true);
-            $file->setPhysicalFilePath($newFilePath);
+            $markAsDeleted = new RenameFile($file->getId());
```

Comment:
Maybe the variable name here should be renameMessage?

Reply:
Sure